packages/offix-datastore/src/Model.ts

```diff
@@ -172,15 +172,19 @@ export class Model<T = unknown> {
     if (data.data && data.data.length > 0) {
       data.data
```

Comment:
@wtrocki @Eunovo maybe we can extract this and the filter below to their own functions?
WDYT?

I think in this case it would make the code more readable.

Reply:
It is pointless to filter IMHO. we should have one module that:
- deduplicate
- deals with deletion (removes or simply have field as _deleted)
etc.